### DISCUSSION TITLE
Add feature to be able to download files via irc/dcc

### DISCUSSION THREAD

**Discussion by EtzBetz** [autobrr/autobrr#1768] [Ideas]
Hey,
I'd really like autobrr to add the feature besides indexing over irc, to also download files via irc directly, since there are some networks, which are also handing out files via dcc there. This would be quite useful, since there is no such automated tool yet.

**Comment by Ttfgggf** [comment]
Would like XDCC download as well.

**Comment by gohamstergo** [comment]
not sure if these topics bump or not, but since there hasnt been any message since last year, guess ill add a +1. @zze0s

**Comment by luckylittle** [comment]
+1 here as well
